Pixel Game/Assets/Scripts/Enemy scripts/EnemyHealth.cs

```diff
@@ -28,6 +28,12 @@ protected override void Start()
 
     protected override void Kill()
     {
+
+        if(dataToPassBetweenScenes.currentActivePlayerQuest.questType == Quest.QUESTTYPE.KILL_ENEMIES)
+        {
+            dataToPassBetweenScenes.currentActivePlayerQuest.IncrementKilledEnemies();
+        }
```

Comment:
Did you intend to keep this? `dataToPassBetweenScenes` is not declared so there's a compiler error.

Reply:
Yes I intended to keep it. My bad :)